ddtrace/contrib/integration_registry/registry.yaml

```diff
@@ -0,0 +1,592 @@
+  - protobuf
+  tested_version_min: 3.19.0
+  tested_version_max: 6.30.2
+
+- integration_name: psycopg
+  is_external_package: true
+  dependency_name:
+  - psycopg
+  tested_version_min: 3.2.1
+  tested_version_max: 3.2.3
+
+- integration_name: pylibmc
+  is_external_package: true
+  dependency_name:
+  - pylibmc
+  tested_version_min: 1.6.3
+  tested_version_max: 1.6.3
+
+- integration_name: pymemcache
+  is_external_package: true
+  dependency_name:
+  - pymemcache
+  tested_version_min: 3.4.4
+  tested_version_max: 4.0.0
+
+- integration_name: pymongo
+  is_external_package: true
+  dependency_name:
+  - pymongo
+  tested_version_min: 3.8.0
+  tested_version_max: 4.10.1
+
+- integration_name: pymysql
+  is_external_package: true
+  dependency_name:
+  - pymysql
+  tested_version_min: 0.10.1
+  tested_version_max: 1.1.1
+
+- integration_name: pynamodb
+  is_external_package: true
+  dependency_name:
+  - pynamodb
+  tested_version_min: 5.5.1
+  tested_version_max: 5.5.1
+
+- integration_name: pyodbc
+  is_external_package: true
+  dependency_name:
+  - pyodbc
+  tested_version_min: 4.0.39
+  tested_version_max: 5.2.0
+
+- integration_name: pyramid
+  is_external_package: true
+  dependency_name:
+  - pyramid
+  tested_version_min: 1.10.8
+  tested_version_max: 2.0.2
+
+- integration_name: pytest
+  is_external_package: true
+  dependency_name:
+  - pytest
+  tested_version_min: 4.6.11
+  tested_version_max: 8.3.5
+
+- integration_name: pytest_bdd
+  is_external_package: true
+  dependency_name:
+  - pytest_bdd
+  tested_version_min: 4.1.0
+  tested_version_max: 6.0.1
+
+- integration_name: pytest_benchmark
+  is_external_package: true
+  dependency_name:
+  - pytest_benchmark
+  is_tested: false
+
+- integration_name: redis
+  is_external_package: true
+  dependency_name:
+  - redis
+  tested_version_min: 2.10.6
+  tested_version_max: 5.2.1
+
+- integration_name: rediscluster
+  is_external_package: true
+  dependency_name:
+  - redis-py-cluster
+  tested_version_min: 2.0.0
+  tested_version_max: 2.1.3
+
+- integration_name: requests
+  is_external_package: true
+  dependency_name:
+  - requests
+  tested_version_min: 2.22.0
+  tested_version_max: 2.32.3
+
+- integration_name: rq
+  is_external_package: true
+  dependency_name:
+  - rq
+  tested_version_min: 1.8.1
+  tested_version_max: 1.16.2
+
+- integration_name: sanic
+  is_external_package: true
+  dependency_name:
+  - sanic
+  tested_version_min: 20.12.7
+  tested_version_max: 24.6.0
+
+- integration_name: selenium
+  is_external_package: true
+  dependency_name:
+  - selenium
+  is_tested: false
+
+- integration_name: snowflake
+  is_external_package: true
+  dependency_name:
+  - snowflake-connector-python
+  tested_version_min: 2.3.10
+  tested_version_max: 3.12.2
+
+- integration_name: sqlalchemy
+  is_external_package: true
+  dependency_name:
+  - sqlalchemy
+  tested_version_min: 1.2.19
+  tested_version_max: 2.0.40
+
+- integration_name: sqlite3
+  is_external_package: false
```

Comment:
This seems possibly inaccurate

Reply:
sqlite3 is actually a stdlib python package!